### PR TITLE
Added/edited content to about, index and contact

### PR DESCRIPTION
**Pull request by VirtualVictoriaa**
Thank you Eamon for looking this over. Best coder ever

### CODE REVIEW

File: contact.md [hunk 1]
```diff
@@ -4,4 +4,33 @@ title: Contact
...
 ---
 
-You can contact us on our official [Discord server]({{ site.discord_invite }}), or you can find us on [GitHub](https://github.com/{{ site.github_username }})
+You can contact us on our official [Discord server]({{ site.discord_invite }}), or you can find us on [GitHub](https://github.com/btc-raspberrypiclub)
+
+# Contact Us
+## Get in Touch
+Have questions, suggestions, or just want to get in touch with us? We'd love to hear from you! Feel free to reach out via email, social media, or using the contact form below.
+
+<!--- Email: raspberrypiclub@gmail.com -->
```
Comment (Agent-E11) [inline contact.md:13]: Do we have an email? And if we do, should this be uncommented?

File: contact.md [hunk 2]
```diff
@@ -4,4 +4,33 @@ title: Contact
...
+Instagram: @rpiclub_btc
+
+<!--- ## Contact Form
```
Comment (Agent-E11) [inline contact.md:16]: I think this contact form should probably just be removed. Unless you know of a way we can utilize it

File: index.markdown [hunk 3]
```diff
@@ -4,3 +4,49 @@
...
+
+**Social Events**
+In addition to our technical activities, we also organize fun and engaging social events to foster camaraderie and friendship among our members. These events include game nights, movie screenings, pizza parties, and outdoor activities. Social events provide opportunities for members to relax, unwind, and connect with like-minded peers outside of the classroom or workshop setting, strengthening bonds and building lasting friendships within the Raspberry Pi community.
+
+## Get Involved
+Whether you're a complete beginner eager to explore the world of Raspberry Pi or an experienced enthusiast looking to share your knowledge and expertise, everyone is welcome to join the Raspberry Pi Club at BTC! Join us in our journey of discovery, learning, and creativity with Raspberry Pi.
+
+## Membership
+Membership in the Raspberry Pi Club is open to all BTC students interested in exploring the fascinating world of Raspberry Pi. To become a member, simply attend one of our meetings or events and sign up! There are no prerequisites or fees to join.
```
Comment (Agent-E11) [inline index.markdown:44]: The prerequisite to join is actually attending 2 meetings. Also, I don't think it is restricted to BTC students

File: contact.md [hunk 4]
```diff
@@ -4,4 +4,33 @@ title: Contact
...
+**Message:**
+[Your Message]
+
+[Submit Button]-->
+
+## Visit Us
+If you're on campus, feel free to drop by during one of our meetings or events! Check our calendar for upcoming dates, times, and locations.
+
+## Stay Updated
+Don't forget to subscribe to our newsletter or follow us on social media to stay up-to-date with the latest news, events, and announcements from the Raspberry Pi Club.
```
Comment (Agent-E11) [inline contact.md:34]: Do we have a newsletter?

**Review by Agent-E11 — CHANGES_REQUESTED** [review]

File: contact.md (hunk 1 above)
Comment (VirtualVictoriaa) [reply contact.md:13]: I was talking about it with Garth and Austin, but we will try to get it working. It is in progress tho

File: contact.md (hunk 2 above)
Comment (VirtualVictoriaa) [reply contact.md:16]: mentioned it to Garth, but we can figure it out this wednesday (3/6)

**Review by Agent-E11 — APPROVED** [review]

**Pull request merged**